### PR TITLE
NR147676 - Postfix : updated dashboard 'About' section

### PR DESCRIPTION
**Pull request by dikshab0202**
Ticket: https://issues.newrelic.com/browse/NR-147676

Updated following points:
1. Image URL on the dashboard has to be updated
2. Link on "Add data" has to be updated to "https://one.newrelic.com/catalog-pack-details?state=a0095d6c-1452-1081-3565-31bcf371c00f"
3. Link on troubleshoot here has to be updated to "https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/flex-integration-tool-build-your-own-integration/"

# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

File: dashboards/sendmail/sendmail.json [hunk 1]
```diff
@@ -284,7 +284,7 @@
...
+            "text": "**About**\n\nInstrument your application with New Relic - [Add Data](https://one.newrelic.com/catalog-pack-details?state=a0095d6c-1452-1081-3565-31bcf371c00f)\n\nUnable to find data in your dashboard? - [Troubleshoot here](\nhttps://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/flex-integration-tool-build-your-own-integration/)\n\nFollow New Relic [Sendmail documentation](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/sendmail-monitoring-integration) to instrument Sendmail.\n\n[Please rate this dashboard](https://docs.google.com/forms/d/e/1FAIpQLSclR38J8WbbB2J1tHnllKUkzWZkJhf4SrJGyavpMd4t82NjnQ/viewform?usp=pp_url&entry.1615922415=Sendmail) here and let us know how we can improve it for you.\n"
```
Comment (mdumpati) [inline dashboards/sendmail/sendmail.json:287]: @dikshab0202 Please change the URLs and text to Postfix

**Review by mdumpati — COMMENTED** [review]
@dikshab0202 check my comments

**Review by rahulbasu-nr — APPROVED** [review]
Looks good to me.

**Review by mdumpati — COMMENTED** [review]
@dikshab0202 Can you remove `sendmail.json` file from this postfix PR

**Review by pkosanam — APPROVED** [review]
LGTM

**Review by seemantk — APPROVED** [review]

**Review by mdumpati — APPROVED** [review]
LGTM

**Pull request merged**